modules/task/task.go

```diff
@@ -23,31 +23,21 @@ package task
 import (
 	"sync"
 
+	"github.com/uber-go/tally"
+
 	"go.uber.org/fx/modules"
-	"go.uber.org/fx/modules/task/internal/stats"
 	"go.uber.org/fx/service"
-
-	"github.com/uber-go/tally"
 )
 
-type globalBackend struct {
-	backend Backend
-	sync.RWMutex
-}
-
 var (
-	_globalBackendMu sync.RWMutex
-	_globalBackend   Backend = &NopBackend{}
-	_asyncMod        service.Module
-	_asyncModErr     error
-	_once            sync.Once
+	_globalBackendMu          sync.RWMutex
+	_globalBackend            Backend = &NopBackend{}
+	_globalBackendStatsClient         = newStatsClient(tally.NoopScope)
```

Comment:
We seem to be using the NoopScope throughout, are we ever using the scope initialized by UberFx?